CMakeLists.txt

```diff
@@ -150,6 +157,17 @@ if(ROBOTOLOGY_ENABLE_TELEOPERATION)
   find_or_build_package(walking-teleoperation)
 endif()
 
+# Human Dynamics Estimation
+if(ROBOTOLOGY_ENABLE_HUMAN_DYNAMICS)
+  set(ROBOTOLOGY_ENABLE_ICUB_HEAD TRUE)
```

Comment:
I am not sure if we already discussed during  the review, but the idea behind profiles is that they are completly independent, and if a user need to set `ROBOTOLOGY_ENABLE_ICUB_HEAD` to `TRUE` on a given machine (for example because he need to connect to iCub CAN devices) he/she can do it manualy. If instead any project in `ROBOTOLOGY_ENABLE_HUMAN_DYNAMICS` depends for compilation on a given project/option, this should be added as all the other dependencies.

Reply:
There is not the compile-time dependency. What if we set `ROBOTOLOGY_ENABLE_ICUB_HEAD` to `TRUE` and  `find_or_build_package(forcetorque-yarp-devices)`, if `ROBOTOLOGY_USES_ESDCAN ` is `TRUE`?  Since with this option the user is indicating, he/she wants to use FT sensors.

Reply:
@traversaro